_data/retail.yml

```diff
@@ -153,6 +153,14 @@ websites:
       - software
     doc: https://help.etsy.com/hc/en-us/articles/115015569567
 
+  - name: EVENTIM
+    url: https://www.eventim.de/
+    twitter: eventimDE
+    facebook: eventimDE
+    email_address: info@eventim.de
```

Comment:
Have you verified that the email address works and is monitored?

Reply:
I have not actually tried it, but it is listed on the imprint page: https://www.eventim.de/help/imprint/